public/locales/en/translation.json

```diff
@@ -343,7 +343,8 @@
         "amazon": "Amazon Login",
         "epic": "Epic Games Login",
         "gog": "GOG Login",
-        "message": "Login with your platform. You can login to more than one platform at the same time."
+        "message": "Login with your platform. You can login to more than one platform at the same time.",
+        "old-mac": "Your MacOS version is {{version}}. MacOS 12 or newer is required to log in the stores."
```

Comment:
suggestions for this copy are welcomed

Reply:
Small pedantic note: the proper capitalization is "macOS".

Reply:
updated, thanks